hpqtypes.cabal

```diff
@@ -140,6 +140,7 @@ library
                      , containers        >= 0.5.0.0
                      , exceptions        >= 0.6
                      , text-show         >= 2
+                     , uuid              >= 1.3      && < 1.4
```

Comment:
You only need to depend on `uuid-types` here, right?

Reply:
You are probably right.